content/plugin/mux/combining-protocol-version-5-providers.mdx

```diff
@@ -0,0 +1,121 @@
+---
+page_title: 'Plugin Development - Combining Protocol Version 5 Providers'
+description: >-
+  Use the tf5muxserver package in terraform-plugin-mux to combine protocol version 5 providers.
+---
+
+# Combining Protocol Version 5 Providers
+
+The [`tf5muxserver`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-mux/tf5muxserver) package enables combining any number of [protocol version 5 provider servers](../how-terraform-works#protocol-version-5) into a single server.
+
+Use this package to:
+
+* Support multiple development teams across separate codebases.
+* Support multiple provider SDK implementations, including but not limited to, combining [terraform-plugin-sdk/v2](../sdkv2/sdkv2-intro) and [terraform-plugin-framework](../framework) after using [tf6to5server](./translating-protocol-version-6-to-5).
```

Comment:
I'm wondering if the detail about how you have to translate them first is a detail we don't need to mention here. Thoughts?

```suggestion
* Support multiple provider SDK implementations, including but not limited to, combining providres built with the [terraform-plugin-sdk/v2](../sdkv2/sdkv2-intro) and the [terraform-plugin-framework](../framework).
```